flake.nix

```
@@ -414,7 +414,7 @@
 
       defaultPackage = forAllSystems (system: self.packages.${system}.nix);
 
-      devShell = forAllSystems (system:
+      developPackages = forAllSystems (system:
```

```suggestion
      developPackage = forAllSystems (system:
```
Logically this should be a singular as there is only one value per system available.